doc/project/project_roles.rst

```diff
@@ -145,35 +126,23 @@ in addition to those listed for Contributors and Collaborators:
 
 Contributors or Collaborators are promoted to the Maintainer role by adding the
 GitHub user name to one or more ``maintainers`` sections of the
-:ref:`maintainers_file` in the Zephyr repository. Candidates who are neither
-Contributors nor Collaborators must be approved by the TSC before they can
-assume the role of Maintainer.
+:ref:`maintainers_file` in the Zephyr repository.
+
+Candidates who are neither Contributors nor Collaborators must be approved by
```

Comment:
This phrasing was always a bit off, and out of place.
a) Collaborators **are** contributors
b) If we just intend to say that if somebody is not a contributor yet, them becoming directly a maintainer needs TSC approval, that is fair. But we have not done that, and it is contrary with the new plan (i.e. "a subset of people makes the decision"). i.e. somebody who is not a contributor becomes a maintainer by being approved in the same subset of people as any other maintainer.
Meaning, this paragraph should just not have the `who are neither Contributors nor Collaborators`